### ISSUE TITLE
How to uninstall

### ISSUE THREAD

**Issue by sledgep** [raspicamplayer/camplayer#57]
Unable to get this to work.

first off, config.txt does not give me any info on how I should input the camera, other than a URL.  what if I need other options like a login.   No sample ini given.

Next how do I remove this application once its installed.   

Also looks like no config to run in portrait mode.

Thank you

**Comment by cybermaus** [comment]
How to login depends on your camera brand and model. You need to read your camera's documentation, or lacking that, search the internet for people with the same camera and check their URL examples. Search for "rtsp url <camera brand>" 

Many/most camera's use a method where they add it before the IP address. 
Like `rtsp://<user>:<pwd>@192.168.1.10/something.cgi/example/channel=0`

But I also seen at early XM camera's (China brand that is resold a lot under other names) where they have it at the end. 
Like `rtsp://192.168.1.10/ something.cgi/example?user=<user>&pass=<pass>&channel=0`

In other words: it is not this software that dictates how to signon, it is your camera.  Above examples are conceptual, what it needs to look like, no need trying those exact examples.
